python/upt-macports/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                upt-macports
+version             0.2.1
+revision            0
+
+maintainers         nomaintainer
+description         MacPorts backend for upt.
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/mojca/upt-macports
+master_sites        https://test-files.pythonhosted.org/packages/83/89/cf55a2b5568d91375145db0c4ead0bd3b0bee81a76f87782c0b4ca26c220/
```

Comment:
This is when you start to appreciate the wonders of the GitHub PortGroup. With that one there is no need to upload strange test files anywhere, you just fetch from a github repository and you're done, even if there has not been any official release yet (you just use the commit's shasum). See the ports that contain `github.setup`.

Reply:
I uploaded to test-pypi as it was convenient for me to generate Portfile using upt instead of using GitHub port group ;)
Also if I do github then which repo should I use ?

Reply:
At this very moment you could do
```
github.setup Korusuke upt-macports <shasum>
version <version>
```
and then you switch as soon as the commits end up in `macports-gsoc/upt-macports` (or `macports-upt-macports`).